src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -147,15 +150,16 @@ private PEModuleBuilder EmitModule
             get { return _factory.CompilationState.ModuleBuilderOpt; }
         }
 
-        public override BoundNode Visit(BoundNode node)
+        [return: NotNullIfNotNull("node")]
```

Comment:
>NotNullIfNotNull [](start = 17, length = 16)

Removing this annotation.  It is not correct.  A local declaration without an initializer is translated into `null`.